php-checks/src/main/java/org/sonar/php/checks/RequireIncludeInstructionsUsageCheck.java

```diff
@@ -63,4 +63,9 @@ private boolean isExcludedFile() {
     String filename = context().getPhpFile().filename();
     return EXCLUDED_FILES.contains(filename);
   }
+
+  private boolean isAutoloadImport(FunctionCallTree tree) {
+    String call = tree.toString();
+    return call.startsWith("require") && call.endsWith("autoload.php'");
+  }
```

Comment:
Nitpick on the method name: it's not just checking that this is an `autoload.php`, but also that this is a `require` call.
Could be renamed to something like `isAutoloadRequireImport` or similar, but I don't have a strong opinion about it.